src/server/src/routes/job/index.ts

```diff
@@ -390,6 +393,79 @@ router.get(
         }
     }
 );
+
+/******************************************************************************
+ *            GET Request - Get New Job - /api/job/get-new-jobs
+ ******************************************************************************/
+
+interface GetNewJobsRequest extends Request {
+    query: {
+        page: string;
+        numOfItems: string;
+    };
+}
+
+router.get(
+    '/get-new-jobs',
```

Comment:
I thought we agreed the get new jobs api was not going to have pagination

Reply:
No, we agreed that get recommended jobs API does not have pagination, but get new jobs API does.